Automattic/wp-calypso
# Add/growth to comparison page

DON'T SHIP UNTIL DEC 3RD WHEN JETPACK GROWTH IS LAUNCHED

## Proposed Changes

* Add Jetpack Growth to comparison table

## Why are these changes being made?

* With a new bundle, we need to add Growth to the page comparing bundles

## Testing Instructions

1. Checkout this branch locally or use the Calypso Green live link
2. Go to `/features/comparison` in Jetpack Cloud
3. Make sure Growth is added and all the features are accurate
![image](https://github.com/user-attachments/assets/893286da-7d27-4f6b-9fe7-b27b5f40e08b)


## Pre-merge Checklist
